Doc/c-api/typeobj.rst

```diff
@@ -1936,6 +1953,8 @@ and :c:type:`PyType_Type` effectively act as defaults.)
    Tuple containing the expanded set of base types, starting with the type itself
    and ending with :class:`object`, in Method Resolution Order.
 
+   This field should be set to ``NULL`` and treated as read-only.
+   Python will fill it in when the type is :c:func:`finalized <PyType_Ready>`.
```

Comment:
```suggestion
   Python will fill it in when the type is :c:func:`initialized <PyType_Ready>`.
```

Reply:
The `PyType_Ready` docs say “Finalize a type object”. Should that change as well?
Either is fine with me but I think it should be consistent.

Reply:
"Finalize" gives the sense that the object should not be used anymore. IMO "initialize" is more appropriate.

Reply:
Well, the link to PyType_Ready makes things clear.